test/unit/vl-rich-data-table-filter.test.html

```diff
@@ -178,6 +222,70 @@
             expect(searchFilter.querySelector('input').value).to.equal('foobar');
         });
 
+        test('Als het data-vl-filter-closable attribuut wordt gezet, worden de sluit knop en filter container met filter knop getoond', () => {
+            const richTable = fixture('vl-rich-data-table-met-filter-fixture');
+            expect(richTable.__filterToggleContainer.hidden).to.be.true;
+            expect(richTable.__filterCloseButton.hidden).to.be.true;
+
+            richTable.setAttribute('data-vl-filter-closable', '');
+
+            expect(richTable.__filterToggleContainer.hidden).to.be.false;
+            expect(richTable.__filterCloseButton.hidden).to.be.false;
+        });
+
+        test('De filter column kan verborgen en getoond worden', () => {
```

Comment:
Er wordt precies nog niet getest of het filter-closed attribute aangepast wordt als het filtering gedeelte gesloten of geopend werd.

Reply:
Misschien beter die hideFilterColumn en showFilterColumn weglaten? Nu kan die via het attribuut ook gesloten worden...

Reply:
Ik zie precies nog geen asserts op "data-vl-filter-closed"?

Reply:
De clicks werden getest via e2e en het zetten van het attribuut via unit, maar ik zal de clicks ook testen via unit testen dan voor de volledigheid, dan kan ik de asserts op data-vl-filter-closed ook meenemen.